test/resources/resourceConfig.spec.js

```diff
@@ -1,78 +1,103 @@
 /* eslint-disable no-unused-expressions */
-import { expect } from 'chai';
-import {
-  buildEndpoint,
-} from '../../src/resources';
-
-describe('Build endpoint', () => {
-  it('static', () => {
-    const endpoint = 'http://new.shoutem.com';
+import { expect } from "chai";
+import rio from "../../src/rio";
+import { buildEndpoint } from "../../src/resources";
```

Comment:
sorry, can't help but notice double-quotes here :) 
can we please return to singles? This is some linter setup I expect.